RaftRs/consensus/RaftRsUdp.tla

```diff
@@ -0,0 +1,1185 @@
+VARIABLES check_quorum   \* check_quorum variables
+VARIABLE progress        \* The status of each follower's receive log, which is used in receiving append, which contains probe and replicate. Corresponding to raft-rs Progress.state
+VARIABLE inflight        \* Number of letters transmitted during the recording process. Corresponding to raft-rs Progress.int (Inflights)  
+
+
+
+(***************************************************************************
+  Network variables and instance
+ ***************************************************************************)
+\* The network is modelled through these three variables
+VARIABLES netman, 
+          netcmd,
+          msgs
+INSTANCE UdpNetwork WITH NULL_MSG <- Nil,
+    _msgs <- msgs, _netman <- netman, _netcmd <- netcmd
+
+
+(***************************************************************************)
+(* Self manipulated invariants checking                                    *)
+(***************************************************************************)
+VARIABLES inv \* Invariants that guarantee correctness
+
+(***************************************************************************)
+(* Vars groups                                                             *)
+(***************************************************************************)
+serverVars    == <<currentTerm, votedFor, raftState>>
+leaderVars    == <<nextIndex, matchIndex>>
+candidateVars == <<voted_for_me, voted_reject>>
+logVars       == <<log, commitIndex>>
+nodeVars      == <<leader_id, check_quorum, progress, inflight>>
+netVars       == <<netman, netcmd, msgs>>
+noNetVars     == <<serverVars, leaderVars, candidateVars, logVars, nodeVars>>
+vars          == <<noNetVars, netVars, inv>>
+
+
+(***************************************************************************)
+(* State constraints helper                                                *)
+(***************************************************************************)
+CONSTANTS Parameters  \* to control the model scale
+
+GetParameterSet(p)  == IF p \in DOMAIN Parameters THEN Parameters[p] ELSE {}
+
+CheckParameterHelper(n, p, Test(_,_)) ==
+    IF p \in DOMAIN Parameters
+    THEN Test(n, Parameters[p])
+    ELSE TRUE
+CheckParameterMax(n, p) == CheckParameterHelper(n, p, LAMBDA i, j: i <= j)
+
+PrePrune(n, p) == CheckParameterHelper(n, p, LAMBDA i, j: i < j)
+
+
+(***************************************************************************)
+(* Type Ok. Used as a check on writing format                              *)
+(***************************************************************************)
+
+TypeOkServerVars == 
+    /\ currentTerm \in [ Servers -> Nat ]
+    /\ votedFor    \in [ Servers -> Servers \cup {Nil} ]
+    /\ raftState   \in [ Servers -> { Follower, Candidate, Leader } ] 
+
+TypeOkLeaderVars ==
+    /\ nextIndex   \in [ Servers -> [ Servers -> Nat \ {0} ]]
+    /\ matchIndex  \in [ Servers -> [ Servers -> Nat ]]
+
+\* TypeOkCandidateVars ==
```

Comment:
why comment out

Reply:
This is an uncleared comment. I'll clear it.

Reply:
seems it's still not cleared @wego1236